### PR TITLE
Create ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60808](https://cumulocity.atlassian.net/browse/MTM-60808)
Original PR: [7016](https://github.softwareag.com/IOTA/cumulocity-ui/pull/7016)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: remotes from manifest should be listed in plugin list (#7016)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: List remotes from manifest in plugin list
```

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60808
+version: 1020.24.3
+---
+remotes from manifest should be listed in plugin list (#7016)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, remotes defined in the manifest were not displayed in the plugin list in the Cumulocity UI. This change ensures that all remotes from the manifest are now properly listed in the plugin list, providing a complete overview of available plugins. This improvement enhances transparency and makes it easier for users to manage and access the remotes defined in their manifest.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60808
+version: 1020.24.3
+---
+Previously, remotes defined in the manifest were not displayed in the plugin list in the Cumulocity UI. This change ensures that all remotes from the manifest are now properly listed in the plugin list, providing a complete overview of available plugins. This improvement enhances transparency and makes it easier for users to manage and access the remotes defined in their manifest.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:17]: @reey What is meant by plugin list? Do you mean the Plugin tab in the application details? Or something else?

Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:17]: Yes exactly, the Plugins tab on the application details

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:17]:
```suggestion
Previously, application plugins were not displayed on the **Plugin** tab in the application details. This change ensures that all application plugins are now properly listed on the **Plugin** tab, providing a complete overview of available plugins.
```

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: List remotes from manifest in plugin list
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:3]:
```suggestion
title: Available plugins correctly displayed in plugin list
```

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:10]:
```suggestion
    label: Administration
```

File: content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: List remotes from manifest in plugin list
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-3-remotes-from-manifest-should-be-listed-in-plugin-list-7016.md:9]:
```suggestion
  - value: component-0UgqXH1Ys 
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**